kale/loaddata/sampler.py

```diff
@@ -268,3 +284,41 @@ def get(self, n):
         i = self.i
         self.i += n
         return self.array[i : self.i]
+
+
+class DomainBalancedBatchSampler(BalancedBatchSampler):
+    """ BatchSampler - samples n_samples for each of the n_domain.
```

Comment:
`n_domain` is just a number so I think this should be `the n domains`

Reply:
I changed to `n_domains` for consistency with line 124 in `BalancedBatchSampler`